daily-programming/2024-05-02/main0502.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+using namespace std;
+int main(){
+  int a,b;
+  cin >> a >> b;
+  int n=0;
+  while(a%b!=0){
+    a++;
+    n++;
+  }

```

Comment:
`while` 文使わずに実装するには...
```suggestion
  // 配る数は b 個未満
  n = b - (a % b);
```